app/src/main/java/de/westnordost/streetcomplete/osm/cycleway/CyclewayParser.kt

```diff
@@ -94,6 +94,7 @@ private fun parseCyclewayForSide(
     val cycleway = tags[cyclewayKey]
     val cyclewayLane = tags["$cyclewayKey:lane"]
     val isSegregated = tags["$cyclewayKey:segregated"] != "no"
+    val isCyclingOkOnSidewalk = tags["sidewalk$sideVal:bicycle"] == "yes" || tags["sidewalk:both:bicycle"] == "yes"
```

Comment:
At first it seems inconsistent that
- `sidewalk:both:bicycle:signed=yes` doesn't need to be present for a road to parsed as `SIDEWALK_OK`
- but selecting `SIDEWALK_OK` adds that tag
- and for separately mapped footways, `...signed=yes` must always be specified for it to be parsed as `ALLOWED_ON_FOOTWAY`

Is this an oversight?

Reply:
Yeah, back from when it was not about beeing signed explicitly. Fixed that.